.changeset/flat-books-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@apollo/gateway": patch
+---
+
+Current request graph path exposure
```

Comment:
It would be awesome to expand this entry a bit. I'd suggest something along the lines of:
>  Exposes, for each subgraph request, the path in the overall gateway operation at which that subgraph request gets inserted. This path is now available as the `pathInIncomingRequest` field in the arguments of `RemoteGraphQLDataSource.willSendRequest` and `RemoteGraphQLDataSource.didReceiveResponse`.

Reply:
Amazing. thanks!